app/build.gradle.kts

```
@@ -72,4 +73,5 @@ dependencies {
     implementation("io.branch.sdk.android:library:5.9.0")
```

This updates the Branch SDK to the latest version
```suggestion
    implementation("io.branch.sdk.android:library:5.14.0")
```